.env.dev

```diff
@@ -4,6 +4,6 @@ PYTHONPATH=${RMGpy}:${PYTHONPATH}
 POSTGRES_DB=postgres
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
```

Comment:
at some point should these passwords and "secret" keys be removed from github (and changed)?